libs/futures/CMakeLists.txt

```diff
@@ -0,0 +1,74 @@
+# Copyright (c) 2019-2020 The STE||AR-Group
+#
+# SPDX-License-Identifier: BSL-1.0
+# Distributed under the Boost Software License, Version 1.0. (See accompanying
+# file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+cmake_minimum_required(VERSION 3.13 FATAL_ERROR)
+
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
+
+set(futures_headers
+    hpx/futures/future.hpp
```

Comment:
`hpx/future.hpp` ?

Reply:
Unforunately no. `async` and friends should be included through `hpx/future.hpp` as well. It has to go in another module.